.ci/lib.sh

```diff
@@ -459,3 +459,16 @@ calc_qemu_files_sha256sum() {
 
 	sha256sum_from_files $files
 }
+
+# Outputs to stdout the environment variables of the current script.
+print_environment() {
+	local have_env="no"
+	[ -n "$(command -v env)" ] && have_env="yes"
+
+	if [ "${have_env}" = "yes" ]
```

Comment:
can we simplify this?

`if [ -n "$(command -v env)" ]`

Reply:
Definitely.
BTW, How can we trigger the tests?

Reply:
@devimc repushed